### DISCUSSION TITLE
Can we applied functionality Bulk Order in Swell Commerce?

### DISCUSSION THREAD

**Discussion by RushiNCode93** [swellstores/community#135] [Help] [answered]
Hello,
I'm Beginner in Swell, and I want to create a new Store in Swell Commerce,
so my question is,

For **bulk ordering** / reorders, I want to allow our customers to quickly reorder products, based on Size.
also want to display sizes/quantities/pricing in this matrix in lots of places.

or is there any **3rd party extension / plugin / app** available to add this kind of functionality?
please help me with this.

thanks in advance.

**Comment by greph** [comment, marked as answer]
Hi @RushiNCode93 , to view a customer's previous orders, you can use the `swell.account.listOrders()` method in swell-js. From there, you can suggest a product to re-order, or filter all products based on a previously ordered attribute, like size. 
https://developers.swell.is/frontend-api/account-orders
